docker-compose.yml

```diff
@@ -3,6 +3,22 @@ version: '3.1'
 
 services:
 
+  localstack:
+    image: localstack/localstack
+    ports:
+      - "53:53"
+      - "443:443"
+      - "4510-4520:4510-4520"
+      - "4566-4620:4566-4620"
+      - "${PORT_WEB_UI-8080}:${PORT_WEB_UI-8080}"
+    environment:
+      - DEBUG=1
+      - DATA_DIR=/tmp/localstack/data
```

Comment:
Suggestion (non-blocking): We can probably start with non-persistent setup. Would you remove this variable? I don't feel strongly though, making this non-blocking.